tools/emailActivityDigest.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-restricted-syntax */
+
+import { Op } from 'sequelize';
+import mailgun from 'mailgun.js';
+
+import { asyncMap } from 'utils/async';
+import { iterAllCommunities } from 'server/community/queries';
+import { Member, includeUserModel } from 'server/models';
+import { renderDigestEmail } from 'server/utils/email';
+import * as types from 'types';
+
+const mg = mailgun.client({
+	username: 'api',
+	key: process.env.MAILGUN_API_KEY!,
+});
+
+const memberQueryOptions = {
+	include: [includeUserModel({ attributes: ['email'], as: 'user' })],
+	permissions: { [Op.or]: ['admin', 'manage'] },
+	raw: true,
```

Comment:
Can I prod at what `{ raw, nest }` are here for? If it's related to memory concerns, could you add a comment about that?

Reply:
Yeah, memory concerns, although now that I'm using a sliding window I probably don't need them.